### ISSUE TITLE
Updating compressor benchmark results (example: zstd to 1.4.5) on benchmarks overview 

### ISSUE THREAD

**Issue by k00bler** [inikep/lzbench#71]
I can't stop but notice that lxbench uses old(er) or outdated versions of some of the compressors. Can we please update the main benchmarks site at https://github.com/inikep/lzbench ?

I can also do this (run everything and update the readme), but i wanted to understand the protocol for changes of the readme first. Happy to help.

**Comment by inikep** [comment]
I don't think that results at https://github.com/inikep/lzbench are very important. The results depends on CPU, compiler, file type etc. The idea behind `lzbench` is that everybody can do their own tests. I update results usually after a bigger release.
